src/widgets/Tabs/PatientSelect.js

```diff
@@ -155,7 +168,19 @@ const PatientSelectComponent = ({
     if (matchedLocalPatient) {
       selectPatient(matchedLocalPatient);
     } else {
-      ToastAndroid.show('No patient found via QR code', ToastAndroid.LONG);
+      // Do a remote search
+      withLoadingIndicator(async () => {
+        try {
+          const remotePatient = await lookupRemotePatient({ barcode: data });
+          if (remotePatient.length) {
+            selectPatient(remotePatient[0]);
+          } else {
+            throw new Error('No patient found via QR code');
```

Comment:
Think this needs to be localised since we're using it for user feedback?

Reply:
Thanks, yeah probs a good idea! 228d105